test/string_view_test1.cpp

```diff
@@ -29,15 +29,24 @@ void interop ( const std::string &str, string_view ref ) {
 void null_tests ( const char *p ) {
 //  All zero-length string-refs should be equal
     string_view sr1; // NULL, 0
-    string_view sr2 ( NULL, 0 );
-    string_view sr3 ( p, 0 );
+    string_view sr2 ( NULL, std::size_t(0) );
```

Comment:
Requiring cast to `size_t` to remove ambiguity will break existing code, not just break the tests.